sklearn/linear_model/ridge.py

```diff
@@ -1014,6 +1014,7 @@ def fit(self, X, y, sample_weight=None):
 
         if error:
             best = cv_values.mean(axis=0).argmin()
+            best_score = cv_values.mean(axis=0)[best]
```

Comment:
please avoid recalculating this mean.

Also, I wonder if we need to follow the `scoring` convention that says errors should be negated so that greater is better. Perhaps in documenting `best_score_` we should be clear that if `scoring is None` we get an error that is being minimised and otherwise, as with `*SearchCV.best_score_` it is maximised.